manage-data/data-store/data-streams/tsds-ingest-otlp.md

```diff
@@ -0,0 +1,86 @@
+---
+navigation_title: "Ingest OTLP metrics"
+applies_to:
+  stack: preview 9.2
```

Comment:
I understand this applies to self-hosted / Stack only, not Serverless, right? And it wouldn't be used by ECH users either as they have mOTLP, is that correct?

Reply:
Technically, the endpoint is available on serverless and ECH, too. But we'd want users to send data to mOTLP for Elastic Cloud.

Reply:
I would just omit the mention of Serverless so that the document is focused (applicable to) Self-managed.